packages/cactus-plugin-bungee/src/main/typescript/web-services/client-endpoint.ts

```diff
@@ -0,0 +1,103 @@
+import type { Express, Request, Response } from "express";
+
+import {
+  IWebServiceEndpoint,
+  IExpressRequestHandler,
+  IEndpointAuthzOptions,
+} from "@hyperledger/cactus-core-api/";
+
+import {
+  Logger,
+  Checks,
+  LogLevelDesc,
+  LoggerProvider,
+  IAsyncProvider,
+} from "@hyperledger/cactus-common";
+
+import { registerWebServiceEndpoint } from "@hyperledger/cactus-core";
+
+import OAS from "../../json/openapi.json";
+import { PluginBUNGEE } from "../plugin-bungee";
+
+export interface CreateViewEndpointOptions {
+  logLevel?: LogLevelDesc;
+  bungee: PluginBUNGEE;
+}
+
+export class ClientEndpointV1 implements IWebServiceEndpoint {
```

Comment:
@eduv09 The name of the endpoint should be based on a verb, e.g. "DoSomethingEndpointV1" instead of "ThingEndpointV1" (where Thing = 'Client' in the specific context).
Please rename it to `CreateViewEndpointV1` (I looked at the code of what it does and that seems to sum it up pretty well).

P.S.: I just saw that you also have an inconsistent naming issue where the endpoint class constructor options are called `CreateViewEndpointOptions`. This is a step in the right direction because there it is a verb, but still, we need to make sure that the options interface and the class name itself are consistent so please make sure to streamline in like this:
`CreateViewEndpointV1Options` and `CreateViewEndpointV1`